packages/nx-playwright/src/executors/playwright-executor/schema.d.ts

```diff
@@ -7,6 +7,7 @@ export interface PlaywrightExecutorSchema {
   headed?: boolean;
   reporter?: string;
   browser?: 'chromium' | 'firefox' | 'webkit' | 'all';
+  runner?: 'yarn' | 'npx' | 'pnpm';
```

Comment:
my thought is an added flag is added contract/guarantee.

any consideration of instead going the [same route as nx-cypress](https://github.com/nrwl/nx/blob/master/packages/cypress/src/executors/cypress/cypress.impl.ts#L13), where the local dependency is utilized?  might even save some startup time :man_shrugging:.

Reply:
Seems like this isn't possible yet - https://github.com/microsoft/playwright/issues/8027

Reply:
cool - thanks for the insight!